src/components/form-content.js

```diff
@@ -94,29 +132,56 @@ const FormContent = ({ history }) => {
     <Form onSubmit={onSubmit}>
       <FieldContainer>
         <Fieldset>
-          <Legend>First oponent</Legend>
+          <Legend>First fighter</Legend>
           <PokemonListContainer>
-          {result.map(pokemon => (
-            <Fragment key={pokemon.name}>
-              <input type="radio" onChange={handleFirst} id={`oponent1-${pokemon.name}`} value={pokemon.name} name="oponent1" />
-              <label htmlFor={`oponent1-${pokemon.name}`}>{pokemon.name}</label>
-            </Fragment>
-          ))}
+            {result.map((pokemon, index) => (
```

Comment:
This component should be split in severals

Reply:
absolutely!